teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,28 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
+    val files: List<ChangeFile>
+}
+
+interface ChangeFile {
+    val fileRevisionBeforeChange: String?
+    val fileRevisionAfterChange: String?
+    val changeType: ChangeType
+    val filePath: String?
+    val relativeFilePath: String?
+}
+
+enum class ChangeType {
+    EDITED,
+    ADDED,
+    REMOVED,
+    COPIED,
+    UNCHANGED,
```

Comment:
`ChangeType.UNCHANGED` looks like an oxymoron :) Why TeamCity may include such changes? If there is no good reason for that, maybe we can skip such files in the library?

Reply:
Looking into TC sources UNCHANGED is the same as UNKNOWN in this context, so I'd get rid of one of those two.